apps/email/js/cards/message_list.js

```diff
@@ -1398,6 +1398,8 @@ return [
                                 sendState === 'sending');
       syncNode.classList.toggle('msg-header-syncing-section-error',
                                 sendState === 'error');
+      // Set the accessible label for the syncNode.
+      mozL10n.setAttributes(syncNode, 'message-header-state-' + sendState);
```

Comment:
Since only outbox messages have a sendState, then ends up being 'message-header-state-undefined' most of the time, and in the logcat, the l10n code prints messages like:

    L10nError: "message-header-state-undefined" not found in en-US in app://email.gaiamobile.org/index.html

So I would like to see something like where the sendState is set to something like this:

    (message.sendStatus && message.sendStatus.state) || 'none'

and then the l10n property file can have a key entry for `message-header-state-none` which can just be a blank string.